packages/@react-spectrum/picker/stories/Picker.stories.tsx

```diff
@@ -55,552 +70,317 @@ let withSection = [
-        <Item key="Three">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'validationState: invalid',
-    () => (
-      <Picker label="Test" validationState="invalid" onSelectionChange={action('selectionChange')}>
-        <Item key="One">One</Item>
-        <Item key="Two">Two</Item>
-        <Item key="Three">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'isQuiet',
-    () => (
-      <Picker isQuiet label="Test" onSelectionChange={action('selectionChange')}>
-        <Item key="100">One hundred</Item>
-        <Item key="2012">Two thousand and twelve</Item>
-        <Item key="3">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'isQuiet, isDisabled',
-    () => (
-      <Picker label="Test" isQuiet isDisabled onSelectionChange={action('selectionChange')}>
-        <Item key="One">One</Item>
-        <Item key="Two million">Two million</Item>
-        <Item key="Three">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'isQuiet, labelAlign: end',
-    () => (
-      <Picker label="Test" isQuiet labelAlign="end" onSelectionChange={action('selectionChange')}>
-        <Item key="One">One</Item>
-        <Item key="Two dollary-doos">Two dollary-doos</Item>
-        <Item key="Three">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'isQuiet, labelPosition: side',
-    () => (
-      <Picker label="Test" isQuiet labelPosition="side" onSelectionChange={action('selectionChange')}>
-        <Item key="One">One</Item>
-        <Item key="Two">Two</Item>
-        <Item key="Three">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'isQuiet, isRequired',
-    () => (
-      <Picker label="Test" isQuiet isRequired onSelectionChange={action('selectionChange')}>
-        <Item key="One">One</Item>
-        <Item key="Two">Two</Item>
-        <Item key="Three">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'isQuiet, isRequired, necessityIndicator: label',
-    () => (
-      <Picker label="Test" isQuiet isRequired necessityIndicator="label" onSelectionChange={action('selectionChange')}>
-        <Item key="One">One</Item>
-        <Item key="Two">Two</Item>
-        <Item key="Three">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'isQuiet, optional, necessityIndicator: label',
-    () => (
-      <Picker label="Test" isQuiet necessityIndicator="label" onSelectionChange={action('selectionChange')}>
-        <Item key="One">One</Item>
-        <Item key="Two">Two</Item>
-        <Item key="Three">Three</Item>
-      </Picker>
-    )
-  )
-  .add(
-    'isQuiet, validationState: invalid',
-    () => (
-      <Picker label="Test" isQuiet validationState="invalid" onSelectionChange={action('selectionChange')}>
-        <Item key="One">One</Item>
-        <Item key="Two">Two</Item>
-        <Item key="Three">Three</Item>
-      </Picker>
+export type PickerStory = ComponentStoryObj<typeof Picker>;
+
+export default {
+  title: 'Picker',
+  component: Picker,
+  excludeStories: [],
+  args: {
```

Comment:
Do we want to add props we weren't testing before like `direction`, `shouldFlip`?